utility/templating.py

```diff
@@ -142,3 +142,10 @@ def load_yaml_to_dict(file):
 def dump_dict_to_temp_yaml(data, temp_yaml):
     with open(temp_yaml, 'w') as yaml_file:
         return yaml.dump(data, yaml_file)
+
+
+def get_crd_dict(path_to_dict):
```

Comment:
We'll want some docs here. Also, this is a nitpick but I'm wondering if we need to not include ``dict`` in the name of this function? Do we always get a dictionary back? Also, ``path_to_dict`` isn't really representative here of what we're giving. It's a path to a yaml file.

Reply:
removed this func based on @clacroix12 comment below